Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -841,6 +841,12 @@ script:
       - sn_si_incident
     description: Deletes a ticket from ServiceNow.
     name: servicenow-delete-ticket
+  - arguments:
+    - description: Attachment File ID.
+      name: file_sys_id
+      required: true
+    description: Delete an attachment from a ticket.
```

According to the design its:
```suggestion
    description: Deletes an attachment from a ticket.
```

---

Judith asked to delete it